### PR TITLE
Include SSO Session duration documentation

### PR DESCRIPTION
**Pull request by jameshill**
In https://github.com/buildkite/buildkite/pull/10577 we created UI to configure the SSO Session Duration. 
The interface is currently behind a feature flag pending deploy of this documentation.

### CODE REVIEW

File: pages/integrations/sso.md.erb [hunk 1]
```diff
@@ -54,6 +54,26 @@ If you are the administrator of an organization within Buildkite with an existin
...
+You can configure the SSO Session Duration to timeout after a predetermined time.
+
+When the specified duration elapses, the user will be signed out of the session.
+
+To set the Session Duration you first need to select the SSO Provider you would
+like to configure.
+
+<%= image "session_duration/select_provider.png", width: 1201/2, height: 786/2, alt: "Screenshot of the Buildkite SSO Settings Page" %>
+
+Then click "Update Session Duration" from the Session Duration section of the
```
Comment (olyism) [inline pages/integrations/sso.md.erb:68]: Please use italics when [referring to a UI elements](https://github.com/buildkite/docs/blob/main/styleguide/STYLE.md#referring-to-ui-elements).

```suggestion
Then click *Update Session Duration* from the *Session Duration* section of the
SSO Provider settings page.
```

File: pages/integrations/sso.md.erb [hunk 2]
```diff
@@ -54,6 +54,26 @@ If you are the administrator of an organization within Buildkite with an existin
...
+SSO Provider settings page.
+
+<%= image "session_duration/update_session_duration.png", width: 1201/2, height: 1043/2, alt: "Screenshot of the Buildkite SSO Settings Page" %>
+
+You can configure the session duration to any timeout between 6 hours and 8760 hours (1 year).
```
Comment (olyism) [inline pages/integrations/sso.md.erb:73]: Please use comma to [separate thousands](https://github.com/buildkite/docs/blob/main/styleguide/STYLE.md#writing-numbers).

```suggestion
You can configure the session duration to any timeout between 6 hours and 8,760 hours (1 year).
```

**Review by olyism — CHANGES_REQUESTED** [review]
Thanks very much for putting this together 🙌  The screenshots look great and the instruction is very succinct 🤩 

Two comments from my end:

1. Are we able to set session duration with GraphQL? If so, could we provide the instructions on the [GraphQL page](https://buildkite.com/docs/integrations/sso/sso-setup-with-graphql)?
2. Some minor style tweaks (see suggestions):
  1. Italicize mentions of UI elements
  2. Thousands separator comma

Cheers :)

**Comment by jameshill** [comment]
> Are we able to set session duration with GraphQL? If so, could we provide the instructions on the [GraphQL page](https://buildkite.com/docs/integrations/sso/sso-setup-with-graphql)? By the way, this is more of a question than a request for change. If we feel users could benefit by raising the awareness of this configuration in GraphQL then we could later think about improving this.

Yes! We could definitely link to the GraphQL documentation ... I was thinking we could link to the Cookbook https://buildkite.com/docs/apis/graphql/graphql-cookbook#update-the-default-sso-provider-session-duration would that be suitable?

File: pages/integrations/sso.md.erb [hunk 3]
```diff
@@ -54,6 +54,26 @@ If you are the administrator of an organization within Buildkite with an existin
 
 If you'd like to have some help with the migration, contact support@buildkite.com.
 
+## SSO Session Duration
+
+You can configure the SSO Session Duration to timeout after a predetermined time. When the specified duration elapses, the user will be signed out of the session.
+
+To set the Session Duration you can either use the [GraphQL API](/docs/apis/graphql/graphql-cookbook#update-the-default-sso-provider-session-duration) or complete
```
Comment (olyism) [inline pages/integrations/sso.md.erb:61]: Yes, linking it to the cookbook would be handy (I wasn't aware we have this content already). Thanks for adding this in.

**Review by olyism — APPROVED** [review]
LGTM 🚀

**Pull request merged**